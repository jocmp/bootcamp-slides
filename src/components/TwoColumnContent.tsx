import * as React from 'react';
import { TwoColumnContentProps } from '../Props'

const TwoColumnContent = (props: TwoColumnContentProps) => (
    <div className="slide-column">
        {props.content}
    </div>
);

export default TwoColumnContent;