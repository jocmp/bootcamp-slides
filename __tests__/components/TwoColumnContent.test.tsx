import * as React from 'react'
import TwoColumnContent from '../../src/components/TwoColumnContent'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('Two-Column Content', () => {

    it('renders', () => {
        const wrapper = shallow(<TwoColumnContent content={"Test content"} />)
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});