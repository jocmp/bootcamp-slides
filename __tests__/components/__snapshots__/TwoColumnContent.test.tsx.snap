// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Two-Column Content renders 1`] = `
<div
  className="slide-column"
>
  Test content
</div>
`;
